src/core/view/TextView.h

```diff
@@ -49,4 +48,8 @@ class TextView {
      * Sets the font name from Text model
      */
     static void updatePangoFont(PangoLayout* layout, const Text* t);
+
+private:
+    const Text* text;
+    static int DPI;
```

Comment:
This should be part of the settings class. We should not duplicate this info within the application.

Reply:
See above